grafana/src/main/resources/application.properties

```diff
@@ -32,4 +32,4 @@ isDownSampling=true
 interval=1m
 # aggregation function to use to downsampling the data
 # COUNT, FIRST_VALUE, LAST_VALUE, MAX_TIME, MAX_VALUE, AVG, MIN_TIME, MIN_VALUE, NOW, SUM
-function=avg
\ No newline at end of file
+function=AVG
```

Comment:
add a comment here: if the data type does not support AVG, then FIRST will be used.

Reply:
done.